usmqe/usmmail.py

```diff
@@ -0,0 +1,90 @@
+"""
+Module for working with email alerts.
+
+.. moduleauthor:: ebondare
+
+Usage::
+
+    import usmqe.usmmail
+
+    # ...get all  messages from /var/mail/root on the client machine
+    # Return a string
+
+    messages = usmqe.usmmail.get_client_mail()
+
+    # ...get messages from /var/mail/root on the client machine
+    # that came within the specified time interval
+    # Return a mailbox object
+
+    messages = usmqe.usmmail.get_msgs_by_time(start_timestamp, end_timestamp)
+"""
+
+
+import pytest
+import usmqe.usmssh
+import mailbox
+import time
+import email.utils
+import os
+import tempfile
+
+
+LOGGER = pytest.get_logger('usmmail', module=True)
+
+
+def create_mailbox_file(filename='mbox_file', content=''):
+    tmpdirname = tempfile.mkdtemp()
+    mbox_file_path = os.path.join(tmpdirname, filename)
+    with open(mbox_file_path, 'w') as f:
+        f.write(content)
+    return mbox_file_path
+
+
+def get_client_mail(host=None, user="root"):
+    """
+    Read mail from /var/mail/root on client machine.
+    Return the contents of the mailbox as a string
+    """
+    SSH = usmqe.usmssh.get_ssh()
+    if host is None:
+        host = usmqe.inventory.role2hosts("usm_client")[0]
```

Comment:
With current change of configuration this line should be changed to:
```suggestion
        CONF.inventory.get_groups_dict()["usm_client"][0]
```
and at the beginning of this file should be:
```
from usmqe.usmqeconfig import UsmConfig
```
and
```
CONF = UsmConfig()
```
as seen for example in https://github.com/usmqe/usmqe-tests/blob/master/usmqe/gluster/gluster.py